### PR TITLE
Fix GoogleAppMeasurement SPM CI

### PR DESCRIPTION
**Pull request by paulb777**
I added a script in #8426 for GoogleAppMeasurement SPM CI but invoked it too late. This should fix it.

Also, do the version bump for 8.7.0 for GoogleAppMeasurement

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -149,7 +149,7 @@ let package = Package(
       url: "https://github.com/google/GoogleAppMeasurement.git",
       // Please keep the version specification aligned with
       // scripts/setup_spm_test_app_measurement.sh.
-      .exact("8.3.1")
+      .exact("8.7.0")
```
Comment (ncooke3) [inline Package.swift:152]: nit: update the above comment now that `scripts/setup_spm_test_app_measurement.sh` is being deleted

**Review by ncooke3 — COMMENTED** [review]
LGTM on green

**Review by ncooke3 — APPROVED** [review]

**Review by htcgh — APPROVED** [review]

File: Package.swift (hunk 1 above)
Comment (paulb777) [reply Package.swift:152]: Good catch! Thanks.

**Comment by AdieOlami** [comment]
I think this is breaking my project as I cannot set up on a new machine. 

Dependencies could not be resolved because no versions of 'GoogleAppMeasurement' match the requirement 8.7.0 and root depends on 'GoogleAppMeasurement' 8.7.0.

**Comment by paulb777** [comment]
We don't generally support working off the master branch. We're currently testing and staging the 8.7.0 release and it's not fully functional externally. Instead, you should use a release version as is documented at https://github.com/firebase/firebase-ios-sdk/blob/master/SwiftPackageManager.md.

**Comment by AdieOlami** [comment]
Thanks for the response, however, using version 8.6.1 gives error `Missing package product 'GoogleAppMeasurement'` that was the reason for using master branch which worked prior.

**Comment by paulb777** [comment]
You might need to `Reset Package Caches` with 8.6.1.

**Comment by AdieOlami** [comment]
I had that issue the first time using SPM. I am doing a fresh install now with that version and that error keeps coming up. That was the reason why I opted to use the master branch then.

**Comment by paulb777** [comment]
Hmm, I'm not sure what is going on. Please open an issue with the Xcode version and a reproducible example and we can take a look.

**Comment by AdieOlami** [comment]
Works now after reseting cache.

**Comment by AdieOlami** [comment]
I also just want to say you guys are doing an amazing amazing job.

**Pull request merged**